questions/279/explanation.md

```diff
@@ -1,27 +1,33 @@
 What does the `std::variant` default constructor do? Which of the types does it pick, or is the variant empty? And if it picks one, how is the value initialized?
 
-`std::variant`s default constructor constructs a variant holding the value-initialized value of the first alternative. So it picks `C` as the type and calls the `C::C()` default constructor initializing it with `i==1`, which gets printed.
+`std::variant`s default constructor constructs a variant holding the value-initialized value of the first alternative. So it picks `C` as the type and calls the `C::C()` default constructor initializing it with `i == 1`, which gets printed.
 
-§[variant.ctor]:
+§[variant.ctor]¶3:
 
-> In the descriptions that follow, let *i* be in the range `[0, sizeof...(Types))`, and `Ti` be the *i<sup>th</sup>* type in `Types....`
-
-    constexpr variant() noexcept(see below );
-
-> Effects: Constructs a `variant` holding a value-initialized value of type T<sub>0</sub>.
+> In the descriptions that follow, let *i* be in the range `[0, sizeof...(Types))`, and T<sub>*i*</sub> be the *i*<sup>th</sup> type in `Types`.
+>
+> `constexpr variant() noexcept(see below);`
+>
+> (...)
+>
+> *Effects*: Constructs a `variant` holding a value-initialized value of type T<sub>0</sub>.
 
 Now what does *value-initialized* mean?
 
-§[dcl.init]¶8:
+§[dcl.init.general]¶9.1.1:
 
 > To *value-initialize* an object of type `T` means:
-> — if `T` is a (possibly cv-qualified) class type (Clause 12) with either no default constructor (15.1) or a default constructor that is user-provided or deleted, then the object is default-initialized;
+>
+> (...)
```

Comment:
I don't know how to properly align 9.1 and 9.1.1, so I've omitted 9.1 here (without a dash sign, because the dashes are on different levels).

* `&nbsp; &nbsp;` doesn't indent long lines properly
* `<li style='list-style-type:"— "'>` indents fine, but doesn't render markdown inside (\`T\`) and looks complicated

Reply:
I don't think it matters, what we have here is good enough. But maybe we should refer to just `§[dcl.init.general]¶9` since we're quoting from more than only 9.1.1 in particular?